### ISSUE TITLE
Ask CLI fails to load AWS_ACCESS_KEY_ID from env variables

### ISSUE THREAD

**Issue by jmeyers91** [alexa/alexa-skills-kit-sdk-for-nodejs#531]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

When I run `ask deploy`, I get this error:

```
[Error]: Cannot find the environment variable: AWS_ACCESS_KEY_ID
```

Then I run `echo $AWS_ACCESS_KEY_ID` and it prints my access key id. I've set the access key in my `.zshrc`:

```
export AWS_ACCESS_KEY_ID=AKGAIHFPUXxxxxxxxxxx
```

And I've double and triple checked it's being run (`source ~/.zshrc`).

I also tried setting the variable in the call to `ask deploy`:

```
AWS_ACCESS_KEY_ID=AKGAIHFPUXxxxxxxxxxx ask deploy
[Error]: Cannot find the environment variable: AWS_ACCESS_KEY_ID
```

## Expected Behavior
Should use the access key env variable.

## Current Behavior
Not using the access key env variable.

## Your Environment

* ASK SDK for Node.js used: 1.6.4
* Operating System and version: 10.14.2

## Node.js and NPM Info
* Node.js version used for development: v10.11.0
* NPM version used for development: v6.4.1

**Comment by jmeyers91** [comment]
It appears this bug was introduced in `ask-cli@1.6.2`. I'm able to deploy successfully with `ask-cli@1.6.1`.

**Comment by RonWang** [comment]
Hi @jmeyers91 

Thanks for reporting!

This issue has been fixed in v1.6.5 release. It was introduced back in v1.6.2 so please upgrade your CLI to unblock.

Best regards,
Ron
